src/server/authserver/Main.cpp

```diff
@@ -113,7 +113,25 @@ extern int main(int argc, char** argv)
 
     sLog->outString("%s (authserver)", _FULLVERSION);
     sLog->outString("<Ctrl-C> to stop.\n");
-    sLog->outString("Using configuration file %s.", configFile);
```

Comment:
@ShinDarth I did not remove the line it is in the code:
[Look carefully] (https://github.com/AzerothCore/azerothcore-wotlk/pull/23/commits/61227171a6d47bce1539128b7f89d51d88ac89db#diff-cbc763271de699b2f62e55b0d2755bb5R134)